Resources/Prototypes/Entities/Markers/Spawners/Random/rocks.yml

```diff
@@ -0,0 +1,23 @@
+- type: entity
+  parent: MarkerBase
+  id: RandomOreRock
+  name: random ore rock spawner
+  components:
+  - type: Sprite
+    layers:
+      - sprite: Structures/Walls/rock.rsi
+        state: rock_random
+  - type: RandomSpawner
+    offset: 0
+    prototypes:
+      - WallRockQuartz
+      - WallRockTin
+      - WallRockPlasma
+      - WallRockUranium
+      - WallRock
+      - WallRockGold
+      - WallRockSilver
+    chance: 0.95
```

Comment:
make this follow roughly the current ore chance for world gen 

```yml
- type: weightedRandom
  id: RandomOreDistributionStandard
  weights:
    OreSteel: 10
    OreSpaceQuartz: 8
    OreGold: 2
    OrePlasma: 4
    OreSilver: 1
    OreUranium: 1
    OreBananium: 0.5
```

Reply:
> make this follow roughly the current ore chance for world gen
> 
> ```yaml
> - type: weightedRandom
>   id: RandomOreDistributionStandard
>   weights:
>     OreSteel: 10
>     OreSpaceQuartz: 8
>     OreGold: 2
>     OrePlasma: 4
>     OreSilver: 1
>     OreUranium: 1
>     OreBananium: 0.5
> ```

do I need to divide them approximately or exactly? in the second case, it seems to me, there will be a lot of lines

in the first case, it can be divided into "ordinary ore" (steel, quartz), "rare ore" (gold, silver, uranium), "ultra-rare ore" (bananium) with a chance for each and then combine them in a spawner.
and in the second case, it is to prescribe the chances exactly for each and then combine them in the same way in the spawner, but this will take up more space

Reply:
@Kit0vras The first case is fine, as long as it isn't way off from the normal weights